# Benthos Release Status Message
int64 timestamp
int8 id
float64 range
